types/cross-spawn/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"indent": [true, "tabs"]
```

Comment:
@sandersn has&nbsp;allowed me&nbsp;to&nbsp;set&nbsp;`indent` in&nbsp;`dtslint.json`:&nbsp;https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33478#issuecomment-470728638.